service/src/main/java/uk/gov/hmcts/reform/fpl/components/NoticeOfChangeAnswersConverter.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.fpl.components;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.fpl.model.Applicant;
+import uk.gov.hmcts.reform.fpl.model.Respondent;
+import uk.gov.hmcts.reform.fpl.model.RespondentParty;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+import uk.gov.hmcts.reform.fpl.model.noticeofchange.NoticeOfChangeAnswers;
+
+
+@Component
+public class NoticeOfChangeAnswersConverter {
+    public NoticeOfChangeAnswers convert(Element<Respondent> respondentElement, Applicant applicant) {
+        RespondentParty respondentParty = respondentElement.getValue().getParty();
+
+        return NoticeOfChangeAnswers.builder()
+                .respondentFirstName(respondentParty.getFirstName())
+                .respondentLastName(respondentParty.getLastName())
+                .respondentDOB(respondentParty.getDateOfBirth())
```

Comment:
there's a discussion ongoing about the dob for respondent (if adding a placeholder instead) but that can be looked at later on. 👍